tests/CameraCaptureTest.cpp

```diff
@@ -0,0 +1,42 @@
+/**
+ * @file   CameraCaptureTest.cpp
+ * @brief  Cameraクラスのテスト
```

Comment:
CameraCaptureクラスのテストかな
```suggestion
 * @brief  CameraCaptureクラスのテスト
```